OpenRA.Game/Settings.cs

```diff
@@ -20,26 +20,66 @@ namespace OpenRA
 {
 	public enum MouseScrollType { Disabled, Standard, Inverted }
 
+	public class LaunchArgs
```

Comment:
I'm not sure I get it. Last I heard, OS integration worked on Linux as well.
Not sure I'm too comfortable with this, but as long as it doesn't break Windows integration...